libs/islamic-website/team-page/src/lib/team-page/team-page.tsx

```diff
@@ -17,27 +17,96 @@ const Founders = [
     description:
       'Andrey is a serial entrepreneur and engineer focused on fintech, cybersecurity, and network communication solutions. He has led teams that have built payment services, corporate messengers, blockchain staking services, and financial analysis software.',
     url: 'https://www.linkedin.com/in/brainsmith/',
+    role: 'Co-Founder, Chief Technical Officer',
   },
   {
     image: '/assets/images/founders/alex-malkov.png',
     title: 'Alex Malkov',
     description:
       '5+ years of legal consulting for leading blockchain and fintech platforms. Alex has advised AAVE, Bequant, Scalable Solutions, Nebula and others.',
     url: 'https://www.linkedin.com/in/probono42/',
+    role: 'Co-Founder, Chief Executive Officer',
   },
   {
     image: '/assets/images/founders/mohammed-alkaff-alhashmi.png',
     title: 'Mohammed AlKaff AlHashmi',
     description:
       'Mohammed AlKaff AlHashmi is a computer science engineer with more than 18 years of experience. With this accumulative experience he focuses on Industry 4.0, Artificial Intelligence, machine learning, Industrial automation, and IOTs. He started his career as an Intelligence Network Engineer at Huawei.',
     url: 'https://www.linkedin.com/in/mohammed-alkaff-alhashmi-19688054/',
+    role: 'Co-Founder, Member of the Executive Board',
   },
   {
     image: '/assets/images/founders/hussein-al-meeza.png',
     title: 'Hussein Mohammed Al Meeza',
     description:
       'Hussein Mohammed Al Meeza is an acclaimed award-winning banker with over 45 years of experience spanning the Islamic banking, finance and insurance sectors. A founding member of Emaar properties, Amlak Finance, Emaar Industries & Investments, Emaar Financial services, Dubai Islamic Insurance & Reinsurance Company (AMAN), Nawat, he was named Best Islamic Banking Personality in 2006. His professional career began in 1975 at the Dubai Islamic Bank (DIB), where he spent over 20 years developing the bank’s services, and still serves as Executive Director. As an emerging star of Islamic banking, he is the founder of Al Salam Banks in Sudan, Bahrain and Algeria, where he serves on various positions including Board Member, Vice Chairman and Chairman of the Executive Committee since 2015. He is also the Chairman of Al Salam Bank - Seychelles, Chairman of Top Enterprises L.L.C., Chairman of Lycée Francais Jean Mermoz L.L.C.',
     url: 'http://www.almeeza.com/biography.php',
+    role: 'Co-Founder, Member of the Executive Board',
+  },
+];
+
+const Team = [
```

Comment:
```suggestion
const team = [
```

Reply:
and `Founders` => `founders`.
use first capital letter only for react compnents and classes